tools/debugging/extract_sp_exceptions.sh

```diff
@@ -83,35 +87,47 @@ function search_for_failures {
     mkdir -p "${DESTINATION_DIR}/errors/"
     echo -e "${BOLD}Looking for failures${RESET}"
     scenarios_dir="${DESTINATION_DIR}/scenarios"
-    for scenario in $(ls $scenarios_dir); do
-        print_bold ${scenario}
+
+    for scenario in "${scenarios_dir}"/*; do
```

Comment:
This yields the scenarios with full path (e.g. `/home/karl/01-31-2020/scenarios/pfs9_partial_withdraw`) and not just the filename.